app/src/main/java/org/xwiki/android/sync/activities/UserListAdapter.java

```diff
@@ -32,28 +33,60 @@
 
 import java.util.List;
 
+/**
+ * {@link android.widget.Adapter} which can be used to show {@link SearchResult} as users
+ */
 public class UserListAdapter extends BaseAdapter {
-    private Context mContext;
+    /**
+     * Context which will be used for operations
+     */
+    private final Context mContext;
+    /**
+     * List which currently shows
+     */
     private List<SearchResult> searchResults;
 
-    public UserListAdapter(Context context, List<SearchResult> searchResults) {
+    /**
+     * Standard constructor
+     * @param context Initial context
+     * @param searchResults Initial list
+     */
+    public UserListAdapter(@NonNull Context context, @NonNull List<SearchResult> searchResults) {
         super();
```

Comment:
You don't need this for BaseAdapter constructor to be called.

Reply:
Thank you, but is it really important?

Reply:
Is it important to have it ? :)

Reply:
No :)